scripts/runfabtests.sh

```diff
@@ -173,7 +175,7 @@ function unit_test {
 	local end_time
 	local test_time
 
-	local e=$(is_excluded $test)
+	local e=$(is_excluded $test_exe)
```

Comment:
`"fi_${test}"` isn't right either though. For example, the first test is `fi_av_test` but with the way the test is written if I did `"fi_${test}"` it would expand to `"fi_av_test -d GOOD_ADDR -n 1 -s SERVER_ADDR"` in which case I'd also be pulling the unquoted `$1` trick.